tests/jeos/image_info.pm

```diff
@@ -76,7 +76,9 @@ sub run {
     my $influxdb_server = get_var('INFLUXDB_SERVER');
     my $influxdb_user = get_var('_SECRET_INFLUXDB_USER');
     my $influxdb_pwd = get_var('_SECRET_INFLUXDB_PWD');
-    return if (get_var('CASEDIR') || !($influxdb_server || $influxdb_user || $influxdb_pwd));
+
+    return if (get_var('CASEDIR') =~ m/^sle$|^opensuse$/);
```

Comment:
Shouldn't the code continue as long as `CASEDIR=sle|opensuse`?

Reply:
damn, yes!

Reply:
the problem is that I can't run VRs :D

Reply:
:D let's do a VR on site. If there will be an issue, another PR would fix it